app/src/main/res/values/strings.xml

```diff
@@ -466,5 +466,5 @@
     <string name="icon_pack_choose_title">Icon pack</string>
     <string name="icon_pack_choose_summary">Icon pack used in the app</string>
     <string name="hide_expired_entries_title">Hide expired entries</string>
-    <string name="hide_expired_entries_summary">Expired entries will be hidden</string>
-</resources>
\ No newline at end of file
+    <string name="hide_expired_entries_summary">No outdated entries shown</string>
```

Comment:
How do entries get outdated? Database entries?

Reply:
A database entry expires or is expired if it is set up like this. Expiration constraints can be defined in other KeePass managers. Not necessarily by time.